mod/web_services/views/default/object/api_key.php

```diff
@@ -43,11 +43,27 @@
 	$content .= elgg_format_element('div', ['class' => ['hidden', 'webservices-secret-' . $entity->guid]], $secret);
 }
 
+// imprint
+$imprint = [];
+
+if ($entity->isKeyActive()) {
+	$imprint[] = [
+		'icon_name' => 'check',
```

Comment:
rewrite to use a ternary inline if/else function... saves a lot of lines of code.. can do for both icon_name and content

Reply:
don't agree